mongo/options/updateoptions.go

```diff
@@ -132,8 +132,144 @@ func (uo *UpdateOptionsBuilder) SetUpsert(b bool) *UpdateOptionsBuilder {
 }
 
 // SetLet sets the value for the Let field.
-func (uo *UpdateOptionsBuilder) SetLet(l interface{}) *UpdateOptionsBuilder {
-	uo.Opts = append(uo.Opts, func(opts *UpdateOptions) error {
+func (uo *UpdateOneOptionsBuilder) SetLet(l interface{}) *UpdateOneOptionsBuilder {
+	uo.Opts = append(uo.Opts, func(opts *UpdateOneOptions) error {
+		opts.Let = l
+
+		return nil
+	})
+
+	return uo
+}
+
+// UpdateManyOptions represents arguments that can be used to configure UpdateMany
+// operations.
+type UpdateManyOptions struct {
+	// A set of filters specifying to which array elements an update should apply. This option is only valid for MongoDB
+	// versions >= 3.6. For previous server versions, the driver will return an error if this option is used. The
+	// default value is nil, which means the update will apply to all array elements.
```

Comment:
We don't expect users to interact with the `...Options` structs directly in most cases. We should move the struct field comments to the corresponding setter methods. Leave a comment on each struct field that references the corresponding setter method, like
> See UpdateManyOptionsBuilder.SetArrayFilters for documentation.

This recommendation applies for the new `DeleteManyOptions` also. Note that we will update the pattern for the rest of the options types with [GODRIVER-3327](https://jira.mongodb.org/browse/GODRIVER-3327).